packages/components/src/checkbox/src/CheckboxField.module.css

```diff
@@ -0,0 +1,59 @@
+.hop-CheckboxField  {
+    /* Default */
+    --hop-CheckboxField-description-color: var(--hop-neutral-text-weak); 
+
+    /* Disabled */
+    --hop-CheckboxField-description-color-disabled: var(--hop-neutral-text-disabled); 
+
+    /* Small */
+    --hop-CheckboxField-sm-row-gap: var(--hop-space-stack-xs);
+    --hop-CheckboxField-checkbox-sm-inline-size: 1rem;
+    --hop-CheckboxField-description-sm-margin-inline-start: calc(var(--hop-CheckboxField-checkbox-sm-inline-size) + var(--hop-space-inline-sm));
+
+    /* Medium */
+    --hop-CheckboxField-md-row-gap: var(--hop-space-stack-sm);
+    --hop-CheckboxField-checkbox-md-inline-size: 1.5rem;
+    --hop-CheckboxField-description-md-margin-inline-start: calc(var(--hop-CheckboxField-checkbox-md-inline-size) + var(--hop-space-inline-md));
+
+    /* Internal variable */
+    --description-color: var(--hop-CheckboxField-description-color);
+
+    align-items: flex-start;
+    row-gap: var(--row-gap, var(--hop-space-stack-sm));
+    display: flex;
+    flex-direction: column;
+    inline-size: max-content;
+    justify-content: flex-start; 
+    max-inline-size: 100%;
+}
+
+.hop-CheckboxField,
```

Comment:
Same comment as above, not sure why it's here.

Reply:
because we can't guarantee that the user is using border-box everywhere